pkgs/development/python-modules/cracklib/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, cracklib }:
+
+buildPythonPackage rec {
+  pname = "cracklib";
+  version = "2.9.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "15ivsi7plmhyzgcyblqxsli42iyg0finhnf94dq3699vj10076b9";
+  };
+
+  # No tests available
+  doCheck = false;
```

Comment:
Wouldn't it make sense to enable tests for the case upstream adds tests?

Reply:
The comment was wrong, it does have tests but they depend on a dictionary that is not included in the archive from pypi (probably because it is relatively big).